package.json

```diff
@@ -15,22 +15,21 @@
     "serve": "docusaurus serve",
     "clear": "docusaurus clear",
     "lint": "standard",
-    "lint:fix": "standard --fix",
+    "lint:fix": "standard --fix && prettier --write scripts src versioned_docs versioned_sidebars *.js",
     "generate-providers": "node ./scripts/generate-providers.js"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.6",
-    "@docusaurus/preset-classic": "2.0.0-beta.6",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.0.0-beta.2",
+    "@docusaurus/core": "2.0.0-beta.9",
+    "@docusaurus/preset-classic": "2.0.0-beta.9",
+    "@docusaurus/remark-plugin-npm2yarn": "^2.0.0-beta.9",
     "classnames": "^2.3.1",
     "lodash.times": "^4.3.2",
-    "mermaid.cli": "^0.3.6",
```

Comment:
the cli is not released separately anymore. and mermaid is a direct dep of "remark-mermaid-dataurl"